www/pages/getting-started/index.md

```diff
@@ -6,7 +6,7 @@ index: 3
 ---
 
 ## Introduction
-First off, thank you for taking the time to check out Greenwood!  As a tool, we hope that you find Greenwood to provide a productive and frictionless developer experience for you that results in performant and engaging experiences for your users.  Under the hood Greenwood is using NodeJS, webpack, babel and a lot of other amazing JavaScript tools to power a modern development workflow based on first class support for the modern web like Web Components, FlexBox, CSS Grid, ECMAScript modules, and all your favorite features!
+First off, thank you for taking the time to check out **Greenwood**!  As a tool, we hope that you find Greenwood to provide a productive and frictionless developer experience for you that results in performant and engaging experiences for your users.  Under the hood Greenwood is using the power of the modern web ([and friends !](/docs/tech-stack/)) to drive a modern development workflow that helps you deliver a modern and perfmant applicatiot site for your users... using the web languages you already know.
 
 ## Your First Project
```

Comment:
There are massive margins for top and bottom on heading within the docs but not apparent on this page? Needs to be identical and reasonable gap between sections.